catroid/src/main/res/values-tr/strings.xml

```diff
@@ -1662,7 +1662,7 @@ bekleyin</string>
     <string name="formula_editor_logic_comparison">Karşılaştırma Operatörleri</string>
     <string name="formula_editor_object_general">Genel özellikler</string>
     <string name="formula_editor_object_movement">Hareket özellikleri</string>
-    <string name="formula_editor_device_visual_detection">Görsel sensörler</string>
+    <string name="formula_editor_device_face_detection">Görsel sensörler</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations